.dotnet/src/Custom/Chat/AssistantChatMessage.cs

```diff
@@ -109,10 +122,15 @@ public AssistantChatMessage(ChatCompletion chatCompletion)
 
         Refusal = chatCompletion.Refusal;
         FunctionCall = chatCompletion.FunctionCall;
+        if (chatCompletion.OutputAudio is not null)
+        {
+            OutputAudioReference = new(chatCompletion.OutputAudio.Id);
+        }
         foreach (ChatToolCall toolCall in chatCompletion.ToolCalls ?? [])
         {
             ToolCalls.Add(toolCall);
         }
+        // Note: The internal Audio property will be populated via reprojected Content upon serialization.
```

Comment:
I think this note and the "CUSTOM" comment on the `OutputAudioReference` property are not needed anymore?